packages/frontend/src/js/app/messages/bg/contactPage.json

```
@@ -11,6 +11,7 @@
   "requestHelp": "Поискате помощ",
   "requestInfo": "Поискате информация",
   "requestAccess": "Поискате достъп до платформата",
+  "registerNewsletter": "",
```

```suggestion
  "registerNewsletter": "Абонирайте се за бюлетина",
```

#google-trad en attendant d'avoir mieux et ne pas nous bloquer